aperturedb/cli/transact.py

```diff
@@ -32,6 +32,19 @@ def load_fuse():
 class OutputTypes(str, Enum):
     STDOUT = "stdout"
     MOUNT_COCO = "mount_coco"
+    RAW_JSON = "raw_json"
+
+
+def dump_as_raw_json(client: Connector, transaction: dict, **kwargs):
+    from aperturedb.CommonLibrary import execute_query
+
+    result, response, blobs = execute_query(
+        client=client,
+        query=transaction,
+        blobs=[])
+    print(json.dumps(response, indent=2))
+    for i, blob in enumerate(blobs):
+        print(f"len(blob[{i}]) = {len(blob[i])}")
```

Comment:
Unfortunately, this makes the output non-JSON.

Reply:
addressed it.